src/evaluation_system/api/plugin.py

```diff
@@ -550,7 +551,7 @@ def add_output_to_databrowser(
             drs_config["variable"] = variable
         user_data = DataReader(plugin_output, **drs_config)
         for output_file in user_data:
-            new_file = user_data.file_name_from_metdata(output_file)
+            new_file = user_data.file_name_from_metadata(output_file)  # type: ignore
```

Comment:
Mypy wasn't complaining before. So let's find out why it is now, instead of making it not complain.